backend/.gitignore

```diff
@@ -1,5 +1,6 @@
 # Environment variables
 .env
+connectionstrings.*json
```

Comment:
Don't include connectionstrings configuration files to ensure we don't get secrets.